Components/Queues/Drivers/AsyncQueueDriverInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers;
+
+/**
+ * Interface AsyncQueueDriverInterface
+ * @package Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers
+ */
+interface AsyncQueueDriverInterface extends QueueDriverInterface
+{
+    const DEFAULT_PORT = 5672;
```

Comment:
The async queue driver interface shouldn't know about the default port that the drivers use to connect

Reply:
Ok, the constant was moved to the driver class.